src/vehicle_gui.cpp

```diff
@@ -3250,6 +3250,31 @@ bool VehicleClicked(const Vehicle *v)
 	return _thd.GetCallbackWnd()->OnVehicleSelect(v);
 }
 
+/**
+ * Dispatch a "vehicle group selected" event if any window waits for it.
+ * @param begin iterator to the start of the range of vehicles
+ * @param end iterator to the end of the range of vehicles
+ * @return did any window accept vehicle group selection?
+ */
+bool VehicleClicked(VehicleList::const_iterator begin, VehicleList::const_iterator end)
+{
+	assert(begin != end);
+	if (!(_thd.place_mode & HT_VEHICLE)) return false;
```

Comment:
Probably needs a comment explaining what this means

Reply:
Same as above, this is copied code.  I have no clue what it means right now, but if you really want an answer I could try reading through more code to figure out an answer.